benchmarks/scripts/__init__.py

```diff
@@ -1,4 +1,17 @@
-from .common import log_performance, save_results, annotate_and_adjust, adjust_axes_to_fit_labels
-from .qiskit_circuits import qaoa_ising_ansatz, qcnn_circuit, VQE_ansatz, random_clifford_circuit
-from .cirq_circuits import cirq_prep_select
-from .generate_layouts import generate_tilted_square_coupling_list, generate_heavy_hex_coupling_list
\ No newline at end of file
+from .common import (
+    log_performance as log_performance,
+    save_results as save_results,
+    annotate_and_adjust as annotate_and_adjust,
+    adjust_axes_to_fit_labels as adjust_axes_to_fit_labels,
```

Comment:
It seems strange to me to import everything as its own name. I've never see that before. Any idea why `ruff` is suggesting that?

Reply:
Yes its a bit aggressive here. It comes from the [rule for checking for unused imports](https://docs.astral.sh/ruff/rules/unused-import/). Its more misleading in this case since the intent was to expose these functions at the module level. By aliasing the imports to the same name, ruff considers it "used".

We have the option of disabling linting for this file, or possibly even just that rule for this file, but I decided to go this direction because it avoids missing other lint checks, or some future case where this actually an unused import we'd want to catch. Open to those other directions though if folks prefer.